src/views/Activate/ActivateDisclaimer.js

```diff
@@ -1,12 +1,15 @@
 import React, { useCallback } from 'react';
-import { Grid, H3, P, B, Flex, Text, CheckboxInput } from 'indigo-react';
+import cn from 'classnames';
+import { Grid, H3, B, Text, CheckboxInput } from 'indigo-react';
 
 import View from 'components/View';
 import { useCheckboxInput } from 'components/Inputs';
 import { ForwardButton } from 'components/Buttons';
 import { useLocalRouter } from 'lib/LocalRouter';
 import WarningBox from 'components/WarningBox';
 
+const kTextStyle = 'f5';
```

Comment:
Still a -1 from me on this naming style for constants. I wasn't going to say anything, but you used `BIG_CASE` in your change to PassportVerify, so... (;

Reply:
the pattern I'm going with is that config-level stuff is `CONSTANT_CASE` and local constants that aren't designed to be flags/config are `kConstant` — thoughts?

Reply:
I would personally forego special naming entirely in that case, and just group them directly under the imports consistently.

Naming prefixes just look ugly to me when you're not going 110% Hungarian notation.

Reply:
fixed in the latest